pkgs/data/fonts/glasstty-ttf/default.nix

```diff
@@ -0,0 +1,19 @@
+{ runCommand, fetchurl, lib }:
+runCommand "glasstty-ttf" {
+  meta = with lib; {
```

Comment:
Could you move the meta _after_ the src, please?  For consistency with Nixpkgs convention.